modules/n1ql/pages/n1ql-language-reference/cost-based-optimizer.adoc

```diff
@@ -24,14 +24,15 @@ The cost-based optimizer takes into account the cost of memory, CPU, network tra
 [[overview]]
 == Overview
 
-The _cost-based optimizer_ (CBO) replaces the previous {query-service-architecture}[rules-based N1QL optimizer].
-It can generate a query plan for {select}[SELECT], {update}[UPDATE], {delete}[DELETE], {merge}[MERGE], and {insert}[INSERT INTO with SELECT] queries.
+The _cost-based optimizer_ (CBO) is a tool that enables the Query service to find the most efficient way of fetching data and performing a query.
```

Comment:
It's not a tool in a "product" sense... Its job is to create the most efficient plan to execute the query.  See: https://dzone.com/articles/less-is-more-why-cost-based-optimizer. and https://dzone.com/articles/cost-based-optimizer-for-couchbase-n1ql-sql-for-js

Reply:
Done